frontend/src/responseQuizView/responseQuizView.js

```diff
@@ -0,0 +1,343 @@
+import React from 'react';
+import './responseQuizView.scss';
+// import ReactTooltipDefaultExport from 'react-tooltip';
+import * as PropTypes from 'prop-types';
+import {
+    Button,
+    ToggleButton,
+    ToggleButtonGroup,
+} from 'react-bootstrap';
+
+import { Navbar, Footer, LoadingPage } from '../UILibrary/components';
+
+// This dictionary maps specific error types to their respective error message.
+const ErrorDescriptions = {
+    'backwards': 'The letters are reversed',
+    'verb-conjugation': 'One or more verbs are incorrectly conjugated',
+    'comma-splice': 'Two independent clauses are incorrectly joined by a comma',
+    'run-on': 'There is a run on sentence',
+    'homophone': 'Misuse of a similar sounding word (homophone)',
+};
```

Comment:
I'd recommend not using PascalCase for the naming of this object. PascalCase is generally reserved in JavaScript for components, classes, and interfaces (as [this article ](https://www.robinwieruch.de/javascript-naming-conventions)outlines). This is by no means a hard and fast rule, just a convention among front-end developers that's worth following if you engage with the broader development community.

Reply:
Renamed to "ERROR_DESCRIPTIONS" to prevent from using PascalCase and because the dictionary being created here is a constant.